src/tablo/Cell/Cell.js

```diff
@@ -7,6 +7,8 @@ const { maybe, enums, union } = t;
 
 const propsTypes = {
   data: t.Any,
+  rowData: t.Any,
```

Comment:
It may be an array or anything (Tablo can be used to render a list).
But for the same reason I mentioned below, `t.Any` covers also `undefined`